### DISCUSSION TITLE
SSH freezes when recieving a Tailscale `SSH_USERAUTH_BANNER` packet

### DISCUSSION THREAD

**Discussion by librarianmage** [blinksh/blink#1929] [Support]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

- Blink v17.1.0.854
- iOS 17.3
- using Tailscale SSH configured to [check mode](https://tailscale.com/kb/1193/tailscale-ssh#configure-tailscale-ssh-with-check-mode)

### Describe the bug

When attempting to SSH to my machine over Tailscale, there is no output. When running with `ssh -vvv`, the logs show that after receiving a `SSH_USERAUTH_BANNER` packet, nothing further is output.

When SSH'ing directly over my local network/Tailscale without Tailscale SSH, or when the Tailscale SSH ACL is configured to accept, everything works correctly.

**Comment by iflowerml** [comment]
Hi! 
- Could you please provide the exact output you get when running `ssh -vvv` ? This way we can check where ssh is getting stuck.
- How did you configure your Tailscale? We’ll try replicating to troubleshoot. 

Thanks

**Reply by carloscabanero** [reply]
Moved this as a discussion until we figure out what may be going on.
